spock/mcp/yggdrasil.py

```diff
@@ -57,29 +53,35 @@ def authenticate(self, username=None, password=None, client_token=None):
         if password:
             self.password = password
         payload['password'] = self.password
-        payload['clientToken'] = client_token if client_token else self.client_token
+        payload[
```

Comment:
This can be rewritten as ```payload['clientToken'] = client_token or self.client_token``` but I would prefer not touching anything of this file due to it's imminent refactor.